rest-api-with-express/cypress.config.js

```diff
@@ -0,0 +1,8 @@
+const { defineConfig } = require('cypress');
+
+module.exports = defineConfig({
+  e2e: {
+    baseUrl: 'http://localhost:3000',
```

Comment:
I wish we could change the port using an env var (e.g. `ROMCAL_APP_PORT` or whatever env var name you prefer).

Reply:
asbefore, will do